include/rocksdb/c.h

```diff
@@ -1221,6 +1222,43 @@ extern ROCKSDB_LIBRARY_API void rocksdb_options_set_comparator(
     rocksdb_options_t*, rocksdb_comparator_t*);
 extern ROCKSDB_LIBRARY_API void rocksdb_options_set_merge_operator(
     rocksdb_options_t*, rocksdb_mergeoperator_t*);
+
+/* Event Listener */
+
+typedef struct rocksdb_flushjobinfo_t rocksdb_flushjobinfo_t;
+typedef int rocksdb_flushreason_t;
+
+extern ROCKSDB_LIBRARY_API rocksdb_event_listener_t*
+rocksdb_event_listener_create(void*, void (*destructor_)(void*));
+
+// Takes ownership of the provided rocksdb_event_listener_t
```

Comment:
I feel this might be redundant given the conventions elsewhere, but I find it a helpful reminder.